pages/welcome/sections/augur_login_section.py

```diff
@@ -0,0 +1,114 @@
+from dash import dcc, html
+import dash
+import dash_bootstrap_components as dbc
+import plotly.express as px
+
+layout = html.Div(
+    className="tab_section_container",
+    children=[
+        html.Div(
+            className="card_section_container",
+            children=[
+                html.Div(
+                    className="card_section_description",
+                    children=[
+                        html.H1("Logging in to Augur"),
+                        html.P(
+                            """
+                            8Knot doesn't manage any user data on its own. In fact, there are no 8Knot accounts, persay.
```

Comment:
IMO the first sentence isnt necessary. I feel like the second sentence is concise and gets the full point across

Reply:
++ okay makes sense.